case_studies.md

```diff
@@ -0,0 +1,30 @@
+# Turing Way - case studies for reproducible research
+
+Reproducible research is work that can be independently verified. 
+In practice, it means sharing the data and code that were used to generate published results - yet this is often easier said than done.
+The Turing Way is a guide to reproducible data science that will support students and academics as they develop their code, with the aim of helping them produce work that will be regarded as gold-standard examples of trustworthy and reusable research. 
+It is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way 
+
+We would like to hear your experience in making your research reproducible and things you have learned that you think others should know! 
+Filling this form should take no longer than 5 - 10 minutes depending on the amount of detail you provide.  
+
+As these case studies will be published in the Turing Way handbook, please only include information that you are comfortable for others to read.
+
+## Tell us who you are
+> Free text answer
+
+## Which issue(s) have you encountered regarding reproducibility of your research?
```

Comment:
Could we expand this to "research in your field"? I think issues related to "your research" could be covered by the final question too, and it would also be good to hear about frustrations/successes associated with reproducing other published work.